File: CivitasRuby/lib/operacion_inmobiliaria.rb
module Civitas
  class Operacion_inmobiliaria
    
    def initialize(g,i)
      @num_propiedad = i
      @gestion = g
    end
    
    def get_gestion
      return @gestion
    end
    
    def get_num_propiedad
      return @num_propiedad
    end
    
  end
end

File: CivitasJava/src/civitas/OperacionInmobiliaria.java
package civitas;

public class OperacionInmobiliaria {
    int numPropiedad;
    GestionesInmobiliarias gestion;
    
    public OperacionInmobiliaria(GestionesInmobiliarias g, int i){
        gestion = g;
        numPropiedad = i;
    }
    
    public GestionesInmobiliarias getGestion(){
        return gestion;
    }
    
    public int getNumPropiedad(){
        return numPropiedad;
    }
}
